src/nvim/buffer_defs.h

```diff
@@ -936,8 +934,8 @@ struct matchitem {
  * All row numbers are relative to the start of the window, except w_winrow.
  */
 struct window_S {
-  uint64_t handle;
-  int w_id;                         ///< unique window ID
+  handle_T handle;                  ///< unique identifier for the window
+
```

Comment:
do you mean in another commit? When i tried compiling it was complaining about b_fnum being defined twice.